jira-wip/src/koans/01_ticket/07_derive.rs

```diff
@@ -14,6 +14,8 @@ mod derive {
     /// You can check the code generated by derive macros using `cargo expand`:
     /// https://github.com/dtolnay/cargo-expand
     ///
+    ///     cargo expand -p jira-wip path_to_enlightenment::derive
+    ///
```

Comment:
Can you surround this with ```bash ... ```?

Reply:
I thought about surrounding it with ``` but it requires a text or shell after that, also when reading shell code in comments it would be better to keep it short such that people are able to read more lines in one shot. Do you think I should still do it?

Reply:
I would still prefer to have it wrapped to single it out.